### ISSUE TITLE
Stops at empty result page after clearing AH.

### ISSUE THREAD

**Issue by Momomi-chan** [Masuzu/ZooeyBot#864]
I think the issue starting occurring for me about a day or two ago while farming AH. At the end of the battle, the page loads to an empty result page (Where it tells you there is no loot to collect and gives you the option of clicking on the Quest button.) 
It isn't constantly refreshing (so it's not the looping issue), it just stops there: https://ibb.co/f9XxuJ

This problem seems to be avoided if I hit the auto button when the battle starts. However, at the end of the following battle, it goes back to the empty result page.

When I click on the quest button, it loads to the actual result page.

Other things to note:
- I'm not using Viramate (So it's not a repeat quest issue)
- I'm using Chromium Version 67.0.3369.0 (Zooey doesn't work on my Chrome)
- I've removed the setting for url prediction on Chromium
- I can farm Ex+ using the Event parameters with no issue
- I tried putting another url in the EventPageUrl (Like you would do for Celestial) but it still doesn't work for AH.

Has anyone else experienced this issue? Though it is GW time, I doubt many people would be using this time to farm AH (such is procrastination on my part).
Thank you for your time.

**Comment by StarrySkies178** [comment]
same problem here

**Comment by Masuzu** [comment]
Hello,
Could you please post here the ZooeyBot.ini file that you are using so that I can check if I can reproduce the issue? Thank you!

**Comment by Masuzu** [comment]
Thank you, I will try it later today.
```
[General]
# Possible values: jp, en. Default to en when the string is malformed. Please make sure that your configuration file is encoded using UTF8 if you use non Unicode characters.
Language=en
# ChromeDevToolsWindowDockedOnTheRight should be set to false if your Chrome DevTools console is docked on the bottom (as seen here https://i.imgur.com/bvTOq0g.png), and to true if it is docked on the right
ChromeDevToolsWindowDockedOnTheRight=true
MaxPageLoadDelayInMsBeforeRetry=5000
# Maximum amount of time Zooey will wait for an expected event to occur (for instance, the attack button appearance) before triggering the fallback policy.
MaxTriggerDelayInMsBeforeFallback=5500
# Maximum delay before an action is deemed to have failed due to a lack of response from the GBF servers
MaxResponseDelayInMs=4500
# Maximum number of retry attempts when an action fails (due to a lack of response for instance) before the fallback policy kicks in
MaxNumActionRetries=5
MinWaitTimeInMsAfterRefresh=3000
# The bot will stop after the given parameter no matter what.
TimeLimitInSeconds=7000
UseViramate=false
# Zooey-bot will stop after using the given number of 1/2 AP potions (resets each time you restart Zooey-bot)
MaxNumPotionsToUse=100
# Zooey-bot will start using full elixirs when this parameter is set to true if there is no 1/2 AP potions left. In any case, no more than MaxNumPotionsToUse potions will be used
UseFullElixirsWhenNoRemainingHalfAPPotions=false

# Set this parameter to true to use full elixirs instead of half AP potions in priority
UseFullElixirsFirst=false

# Horizontal position of the countdown timer which appears when a nightmare mode is available, with regards to the top left corner of your desktop. No countdown timer is displayed when the Event mode is activated.
CountdownTimerHorizontalPosition=0
# Vertical position of the countdown timer which appears when a nightmare mode is available, with regards to the top left corner of your desktop. No countdown timer is displayed when the Event mode is activated.
CountdownTimerVerticalPosition=0

# The "slave mode" is used to host Co-Op quests (the slime H2-1 stage for instance) using 2 accounts on a same computer.
# It requires the use of 2 accounts and the "zooey-bot-slave" Chrome extension installed in addition to the "zooey-bot" Chrome extension.
# The "slave account" will host the quests and pay the AP cost.
# You can log into a second account on a same computer using Chrome Incognito mode.
# To use this mode, set the parameter Active to true and follow the console instructions.
[SlaveMode]
Active=false
# Relative path to the script to use to customize the behaviour of the main account.
# If left blank, the slave account will attack at each turn.
MainAccountLuaScript=Scripts/slime_blasting_sword_master.lua
# Relative path to the script to use to customize the behaviour of the slave account.
# If left blank, the slave account will attack at each turn.
SlaveLuaScript=Scripts/slime_blasting_sword_master.lua
# By default, Zooey will make the slave account attack first. Set this parameter to true if you would like your main account to attack first.
ProcessMainAccountTurnFirst=false

[Inputs]
DelayInMsBetweenMouseDownAndUp=40
RandomDelayInMsBetweenMouseDownAndUp=20
MouseSpeed=580
MouseScrollSpeed=250
# The bot will stop when you press the given key. The default key is F1. Please refer to https://msdn.microsoft.com/en-us/library/windows/desktop/dd375731(v=vs.85).aspx (the values on this page are given in hexadecimal base).
# If you also use Viramate, DO NOT use a key which is shared by Viramate shortcuts.
ExitKeyCode=112
# After the bot moves the cursor to the target position, wait for the given duration before performing the click
WaitTimeInMsBeforeClickInput=60

[PartySelection]
# Integer which defines the party group to use. If an integer outside of the range [1, 7] (boundaries included) is provided, the current party group will be selected.
PreferredPartyGroup=0
# Integer which defines the party deck to use. If an integer outside of the range [1, 6] (boundaries included) is provided, the current party deck will be selected.
PreferredPartyDeck=0
# Integer which defines the party group to use for the nightmare mode. If an integer outside of the range [1, 7] (boundaries included) is provided, the current party group will be selected.
PreferredNightmareModePartyGroup=0
# Integer which defines the party deck to use for the nightmare mode. If an integer outside of the range [1, 6] (boundaries included) is provided, the current party deck will be selected.
PreferredNightmareModePartyDeck=0
# Relative path to the Lua script to use to select programmatically a party based on the selected support summon
PartySelectionScript=party_selection.lua

[Summons]
# Comma-delimited list of summons which will be selected in priority if they are available
PreferredSummons=Kaguya,White Rabbit
# In case none of the preferred summons was found, select a random summon in the given summon attribute tab, unless the parameter RerollSummonWhenNoPreferredSummonWasFound is set to true.
# Possible values are 'misc' (Kaguya, White Rabbit fall into this category), 'fire', 'water', 'earth', 'wind', 'light' or 'dark'.
# Not case-sensitive.
DefaultSummonAttributeTab=Dark
# Zooey will reroll summons until one of the preferred summons is available, using trial battles to refresh the summon list. This does not apply to the summon selection for the nightmare stage (see the EventMode section).
RerollSummonWhenNoPreferredSummonWasFound=true

[Combat]
# Zooey-bot will wait at least the given number of milliseconds after pressing the attack button
MinWaitTimeInMsAfterAttack=600
# Zooey-bot will wait at least the given number of milliseconds after using a summon
MinWaitTimeInMsAfterSummon=3300
# Zooey-bot will wait at least the given number of milliseconds after using a skill
MinWaitTimeInMsAfterAbility=2700
# Relative path to the script to use to customize the pace of combats. Embrace a world of infinite possibilities!
# Leave this parameter blank if you do not want to use any script. In this case Zooey will just attack at each turn. You can test your scripts using the TrialBattleMode (see the Debug section).
# Please refer to https://github.com/Masuzu/ZooeyBot/wiki/Lua-scripts for a comprehensive overview of the scripting features.
LuaScript=
# If set to true, Zooey will refresh the page after each attack on the last stage of a quest.
# This saves up some precious seconds and that is what every MVP racer does.
ReloadPageOnLastBattle=true
# Maximum wait time in milliseconds to load the character selection menu when an ability targetting a party member is used.
MaxWaitTimeToLoadCharacterSelectionMenuInMs=1100
EnableEarlyTurnStartWithViramate=false
# This parameter is only taken into account when UseViramate and EnableEarlyTurnStartWithViramate are set to true, which makes it possible to cast skills or use summons before the raid stage is fully loaded. It configures how long to wait before taking any action, from the moment Zooey deemed it possible to start casting skills or using summons.
MinWaitTimeBeforeActionAfterEarlyTurnStartInMs=1500

# After performing a chain burst with 2 or more characters, you can't take any actions for a short period of time.
# Use the parameters below to configure how long you would like to wait when it happens.
AdditionalDelayInMsAfter2ChainBurst=3000
AdditionalDelayInMsAfter3ChainBurst=7000
AdditionalDelayInMsAfter4ChainBurst=11500
AdditionalDelayInMsAfterOverchainBurst=11500

# Showcase: Zooey bot project - Cerberus showdown 3-hour demo - automated event grinding
# https://www.youtube.com/watch?v=-xvDRwB4QEk
# Applicable to any quest with a nightmare mode such as Angel Halo and Xeno showdowns.
# Please refer to https://github.com/Masuzu/ZooeyBot/wiki/Event-mode for more information.
[EventMode]
# Set this parameter to true to use the event mode. Zooey will enter autopilot mode when you navigate to the quest menu.
Enabled=true
# Url of the summon selection page for Angel Halo (Extreme)
EventRaidUrl=http://game.granbluefantasy.jp/#quest/supporter/510031/5
EventRaidScript=
# Url of the summon selection page for Dimensional Halo
NightmareModeUrl=http://game.granbluefantasy.jp/#quest/supporter/510051/5
NightmareModeScript=Scripts/DimensionalHalo2.lua
# Some events such as Rise of the Beasts require to navigate to the home page of the event before navigating to the event raid url defined above. If EventPageUrl is provided, Zooey will first navigate to this url before hosting the raid. Leave EventPageUrl blank for events which do not require you to do so.
EventPageUrl=
# Comma-delimited list of summons which will be selected in priority if they are available for the nightmare mode.
# In case none of the preferred summons was found, select a random summon in the summon attribute tab defined by the parameter DefaultSummonAttributeTab, unless the parameter RerollSummonWhenNoPreferredSummonWasFoundForNightmareMode is set to true.
NightmareModePreferredSummons=Luminiera Omega_1
# Zooey will reroll summons until one of the preferred summons for the nightmare mode is available, using trial battles to refresh the summon list.
RerollSummonWhenNoPreferredSummonWasFoundForNightmareMode=true
# Set this parameter to true if the nightmare mode is available when you start Zooey so she can proceed to the nightmare stage right ahead.
NightmareModeAvailableAtStart=false
# Wait time in milliseconds after Zooey finishes loading the EventPageUrl
WaitTimeInMsAfterEventPageIsLoaded=1000

# Use this special event mode to automate farming of events stages which require a certain number of items to be hosted, such as Guild War nightmare modes.
# Please refer to https://github.com/Masuzu/ZooeyBot/wiki/Treasure-Raid-Event-mode for more information
[TreasureEventMode]
Enabled=false
# Url of the event page. The URL of the June 2017 dark GW is given as an example.
TreasureEventUrl=http://game.granbluefantasy.jp/#quest/extra/event/2018
# Supported value are 'vh' (for "very hard"), 'ex' (for "extreme"), 'impossible' (introduced since the "Right Behind You" event), 'xeno-raid' (for the level 100 Xeno raid), 'gw' (for the preliminary round and first day of "Unite and Fight" nightmare mode level 90), 'gw-day-2-nm-90' for the second and 3rd days of "Unite and Fight" nightmare mode level 90, 'gw-day-2-nm-95' for the second and 3rd days of "Unite and Fight" nightmare mode level 95, 'gw-day-4-nm-90' for the 4th and last days of "Unite and Fight" nightmare mode level 90, 'gw-day-4-nm-95' for the 4th and last days of "Unite and Fight" nightmare mode level 95, 'gw-day-4-nm-100' for the 4th and last days of "Unite and Fight" nightmare mode level 100
Difficulty=gw-day-4-nm-90
# Action point cost of the quest you are trying to start
ActionPointCost=30
TreasureEventModeScript=


NightmareModeUrl=
NightmareModeScript=
# Comma-delimited list of summons which will be selected in priority if they are available for the nightmare mode.
# In case none of the preferred summons was found, select a random summon in the summon attribute tab defined by the parameter DefaultSummonAttributeTab, unless the parameter RerollSummonWhenNoPreferredSummonWasFoundForNightmareMode is set to true.
NightmareModePreferredSummons=Bahamut
# Zooey will reroll summons until one of the preferred summons for the nightmare mode is available, using trial battles to refresh the summon list.
RerollSummonWhenNoPreferredSummonWasFoundForNightmareMode=true
# Set this parameter to true if the nightmare mode is available when you start Zooey so she can proceed to the nightmare stage right ahead.
NightmareModeAvailableAtStart=false

# Use this mode to host and repeat co-op stages on your own.
# More information about this mode can be found here https://github.com/Masuzu/ZooeyBot/wiki/Solo-co-op-mode
[SoloCoOpMode]
Enabled=false
# Relative path to the script to use to customize the behaviour of the main account.
# If left blank, the slave account will attack at each turn.
LuaScript=Scripts/slime_blasting_sword_master.lua

# Use this mode to repeat a certain number of times raid A, then move on to raid B and so on. 
# More information to get started with the customized raid scheduling mode can be found here https://github.com/Masuzu/ZooeyBot/wiki/Customized-raid-scheduling
[CustomizedScheduling]
Enabled=false
SchedulingLuaScript=scheduling.lua
PlaySoundNotificationAfterScheduling=false

[Debug]
Clicks=false
# When this parameter is set to true, Zooey bot won't start until you press ENTER. This is especially useful to test your Lua scripts using trial battles.
TrialBattleMode=false

# With this mode, Zooey will not perform any action. Use this mode to capture status effects by clicking on the list of status effects of characters and/or enemies, as shown here: https://i.imgur.com/bdneWen.png). A summary file named "status_effects.txt" will also log the captured status effects.
[ListenerMode]
Enable=false

[Alerting]
# Relative path to the mp3 or wav file to play when the captcha window appears
CaptchaNotificationSoundPath=captcha_notification.mp3
# Number of times the captcha notification sound is played before Zooey stops, following a captcha appearance.
NumNotifications=9999
# Maximum number of times in a row allowed when Zooey has to refresh the summon page. Reloading an abnormally high amount of times can indicate that the captcha popup is displayed but that Zooey somehow missed it.
# A sound notification will be played when the summon selection page is refreshed more than this number of times. 
MaxNumSummonSelectionFailuresBeforePlayingSoundNotification=5
StopAndPlaySoundAlertOnNightmareModeAppearance=false

[DimensionalHalo]
# It is possible to know in advance whether the Dimensional Halo boss will transform or not.
# Unless you are not already drowning in Angel fodder, set this parameter to true to retreat immediately at the start of the battle if the boss won't transform.
RetreatWhenNoDimensionalHaloTransformation=true
```

**Comment by Masuzu** [comment]
I can't reproduce the issue on my side without Viramate and the same settings.
Could you please show the messages in the console window when you are "stuck"?

**Comment by qwergbf** [comment]
i have same problem
https://i.imgur.com/5YBjRtb.png

**Comment by eichisama** [comment]
Same issue here.

**Comment by Masuzu** [comment]
Oh I get it now. On my side I always stay on the result page when the "Move to result page" message is displayed.

**Comment by agesa** [comment]
Same issue. Reverted to 2.11.8 and it works fine.  Can use that as a temporary workaround and switch back to current when farming something else.

**Comment by yurihanabira** [comment]
![image](https://user-images.githubusercontent.com/39130804/43258676-2596863c-9106-11e8-92e1-
de00b7321413.png)

Same thing with current event

**Comment by lililelee** [comment]
I also have the same issue

**Comment by Masuzu** [comment]
This is now addressed in v2.14.2
